cli/remotebazel/remotebazel.go

```diff
@@ -1026,6 +973,104 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 	return exitCode, nil
 }
 
+func attemptRun(ctx context.Context, bbClient bbspb.BuildBuddyServiceClient, execClient repb.ExecutionClient, req *rnpb.RunRequest) (*inpb.GetInvocationResponse, *repb.ExecuteResponse, error) {
+	childCtx, cancelChildCtx := context.WithCancel(ctx)
+	defer cancelChildCtx()
```

Comment:
nit: to avoid wrangling multiple contexts you could also use a `done` chan and `defer close(done)`

Reply:
When using a defer, this is unneeded